aws-rds-cfn-common/src/main/java/software/amazon/rds/common/handler/Commons.java

```diff
@@ -62,7 +62,7 @@ public static <M, C> ProgressEvent<M, C> handleException(
             // If we not set model and context to null, CFN will attempt to perform delete of the resource it has attempted to create,
             // even if resource belongs to a different stack altogether, or was created out of bounds
             if (handlerErrorStatus.getHandlerErrorCode() == HandlerErrorCode.AlreadyExists) {
-                return ProgressEvent.failed(null, null, handlerErrorStatus.getHandlerErrorCode(), exception.getMessage());
+                return ProgressEvent.failed(null, progress.getCallbackContext(), handlerErrorStatus.getHandlerErrorCode(), exception.getMessage());
```

Comment:
Please update the comment above to reflect the change.

Reply:
Change no longer needed.